docs/build-tools/sbt.md

```diff
@@ -27,6 +27,8 @@ After that, start up sbt or reload your sbt shell to load the plugin.
 
 The sbt command `bloopInstall` exports your sbt build to bloop.
 
+If you have multiple projects, you should run `bloopInstall` in the projects that aren't dependent on other projects first. For example:  acme-core-server references acme-shared-lib.  So in the acme-shared-lib project run `sbt bloopInstall` first and then in the acme-core-server run sbt bloopInstall second.
```

Comment:
Could we move this paragraph after the following paragraph, where we explain how an sbt project is represented?